flow2ml/Filters.py

```diff
@@ -37,6 +37,12 @@ def applyMedian(self,classPath):
       if img is not None:
         # applies median filter to the image.
         median = cv2.medianBlur(img, 5)
+
+        lt=[] #empty list
+
+        lt.append(median) # storing the images in the list
+
+        c=np.stack(lt) # converting the stored iamges in the list into a 3D array which is "c" here
```

Comment:
You must declare variable c as class variable and proceed. Store all processes images into variable “c”
And also give a meaningful name for variable “c”